store/reducers/groups.reducer.js

```diff
@@ -499,6 +704,57 @@ export default function groupsReducer(state = initialState, action) {
         error: action.error,
         loadingActivities: false,
       };
+    case userActions.USER_LOGOUT:
+      return {
+        ...newState,
+        usersContacts: null,
+        geonames: null,
+        peopleGroups: null,
+        search: null,
+      };
```

Comment:
I think this user logout action should be in a different reducer than the groups reducer.

Reply:
I detect this action in the groupsReducer because i need to clean the collections from groupsReducer when the user logout.